### ISSUE TITLE
[Feature request]3rd party plugins

### ISSUE THREAD

**Issue by zaoqi** [koreader/koreader#4405]
There isn't a user-friendly way to install third-party plugins (if they exists).
I have to open a book for open a plugin.

**Comment by zaoqi** [comment]
And l10n will have to be located at somewhere like /plugins/*/...

**Comment by Frenzie** [comment]
I think editing the OP completely makes it a bit hard to follow the conversation.

> There isn't a user-friendly way to install third-party plugins (if they exists).

You can just drop them in plugins.

> I have to open a book for open a plugin.

You don't, though?

> And l10n will have to be located at somewhere like /plugins/*/...

Ah yes, that's a good point.

**Comment by Frenzie** [comment]
> And l10n will have to be located at somewhere like /plugins/*/...

Create a child class using a different GetText.dirname and you're golden.

See https://www.lua.org/pil/16.1.html
